--- golang-github-smartystreets-assertions-1.10.1+ds/internal/go-render/render/render_test.go
+++ golang-github-smartystreets-assertions-1.10.1+ds/internal/go-render/render/render_test.go
@@ -262,10 +262,6 @@
 			"map[bool]struct {}{false:{}, true:{}}",
 		},
 		{
-			map[interface{}]struct{}{1: {}, 2: {}, 3: {}, "foo": {}},
-			`map[interface{}]struct {}{1:{}, 2:{}, 3:{}, "foo":{}}`,
-		},
-		{
 			map[complex64]struct{}{1 + 2i: {}, 2 + 1i: {}, 3 + 1i: {}, 1 + 3i: {}},
 			"map[complex64]struct {}{(1+2i):{}, (1+3i):{}, (2+1i):{}, (3+1i):{}}",
 		},
